### DISCUSSION TITLE
Centred combo element on drum in Taiko

### DISCUSSION THREAD

**Discussion by ashprids** [ppy/osu#28484] [Ideas]
In Stable, the combo counter for Taiko is placed in the centre of the drum.
![image](https://github.com/ppy/osu/assets/39588029/889559c4-bd38-4454-9c0f-14cbd5dc0fbe)
There's no way to implement this for Lazer as far as I'm aware because the only legacy combo element is for standard, so it doesn't have centered text and it has this annoying flashy effect whenever it increments.

It would be nice to have an element or option to centre the combo counter and have its functionality similar to stable somehow, at least for Taiko.

**Comment by Stoppedpuma** [comment]
> There's no way to implement this for Lazer

It's possible by moving it on top of the drum.

![image](https://github.com/ppy/osu/assets/58333920/f64ee0f3-96c3-466b-bb1b-a00985e669a8)
![image](https://github.com/ppy/osu/assets/58333920/16ffd46c-3e3d-47ce-8c82-4c040edbecec)

**Reply by BlockHead007** [reply]
Aaaaand... looks awful with that x at the end

**Reply by Stoppedpuma** [reply]
You do realise you can remove it right?

**Reply by ashprids** [reply]
AFAIK you can only do that by editing the skin files to make the X transparent, which isn't really an answer to the problem. What if I want osu!standard to use that combo object with the X, but not Taiko? 

Even if you place it at the top, it still flashes quite distractingly. And anyways, I feel like having the option to centre text skin elements in the editor would be beneficial for many other purposes.
